ci/common-ci.yml

```diff
@@ -6,6 +6,9 @@ stages:
   - build
   - test
 
+variables:
+  FF_TIMESTAMPS: true
+
```

Comment:
To do: will this be inherited by the pipeline generated by `ctest_to_gitlab.sh`? Probably not, but check it.

Reply:
It won't. Added to `ctest_to_gitlab.sh` as well in https://github.com/eth-cscs/DLA-Future/compare/4234fc91ed6c209be7fd0bcb58a605db28a5d716..22e82b065dbc497ed2b3077a3c1ef6cae76df9a4.

Reply:
Did you try https://github.com/eth-cscs/DLA-Future/blob/ci/alps_1/ci/common-ci.yml#L110-L111 ?

Reply:
I did not! Thanks for pointing that out, I wasn't aware that existed. Trying it out in https://github.com/eth-cscs/DLA-Future/pull/1228/commits/2438e960bad38a256fa981b39e0db760bfc170a8.

Reply:
https://github.com/eth-cscs/DLA-Future/commit/2438e960bad38a256fa981b39e0db760bfc170a8 did not seem to have an effect, but I don't know if I'm using it wrong. Do you know if there are any special requirements on what variables are forwarded?

Reply:
Had a look to the doc:
The correct forward should be `yaml_variables` but should be true by default.
Worth a try explicitly setting it.
https://docs.gitlab.com/ee/ci/yaml/#triggerforward

Reply:
Yep, thanks. Let's see: https://github.com/eth-cscs/DLA-Future/pull/1228/commits/a10b945d6776c0550c5a1cfca1d1f00e3a5fe414.

Reply:
😞 Let's rollback to 22e82b0. Sorry for the noise.

Reply:
No worries, I at least learned about a new gitlab feature :wink: It's now back to the original version.